workflow_array_ephys/ingest.py

```diff
@@ -71,18 +77,22 @@ def ingest_sessions(session_csv_path='./user_data/sessions.csv'):
         session_key = {'subject': sess['subject'], 'session_datetime': min(session_datetimes)}
         if session_key not in session.Session():
             session_list.append(session_key)
-            session_dir_list.append({**session_key, 'session_dir': session_dir.relative_to(root_data_dir).as_posix()})
+            root_dir = element_data_loader.utils.find_root_directory(
+                                                    get_ephys_root_data_dir(), sess_dir)
+            sess_dir_list.append({**session_key, 'session_dir': sess_dir.relative_to(root_dir).as_posix()})
             probe_insertion_list.extend([{**session_key, **insertion} for insertion in insertions])
 
     print(f'\n---- Insert {len(session_list)} entry(s) into session.Session ----')
-    session.Session.insert(session_list)
-    session.SessionDirectory.insert(session_dir_list)
+    # Broz 102821 - prev, skip_dupes was true for ingest_subj, but not ingest_sess
+    #               I thought they should mirror each other, chose both True
+    session.Session.insert(session_list, skip_duplicates=True)
+    session.SessionDirectory.insert(sess_dir_list, skip_duplicates=True)
 
     print(f'\n---- Insert {len(probe_list)} entry(s) into probe.Probe ----')
-    probe.Probe.insert(probe_list)
+    probe.Probe.insert(probe_list, skip_duplicates=True)
```

Comment:
For `probe.Probe`, I don't believe that `skip_duplicates=True` is required because of the following line:
```
if probe_key['probe'] not in [p['probe'] for p in probe_list] and probe_key not in probe.Probe():
    probe_list.append(probe_key)
```

Reply:
Yes, generally, do not use `skip_duplicates`. Catching duplication errors may be important for some integrity checks.

Reply:
Good note! I now see why this isn't needed, thanks. And yes, certainly best practice to leave as default `skip_duplicates=False`